.github/workflows/rubi-ci.yml

```diff
@@ -0,0 +1,68 @@
+# Run this job on PRs and pushes to master
+
+name: CI for rubi-py`
+
+on:
+  push:
+    branches: [ "master" ]
```

Comment:
Also - maybe these checks shouldn't happen when pushing to "master"? 

All tests should be checked when creating a PR for master. But pushing into master should activate a different workflow that actually updates the package itself.

Reply:
since in theory no one should be pushing to master anyways, i think it is fine to remove it (need to revoke my own push to master permissions...). that being said, it shouldn't impede another action being run, so not the worst thing 🤷